README.md

```diff
@@ -6,7 +6,41 @@ An Open Source Social Media Platform and Public Forum for Questions and Discussi
 - **Live Demo: (https://www.mumble.dev/)**
 - **UI Kit (Docs/Preview) 👉 [Here](http://mumble-lp.s3-website-us-west-2.amazonaws.com/)**
 - **Discord [Join here](https://discord.gg/TxgpyK8pzf)**
+---
+## Requirements
 
+For development, you will need Node.js and a node global package, NPM or Yarn, installed in your environement along with a text editor of your choice.
+
+### Node
+- #### Node installation on Windows
+
+  Just go on [official Node.js website](https://nodejs.org/) and download the installer.
+Also, be sure to have `git` available in your PATH, `npm` might need it (You can find git [here](https://git-scm.com/)).
+
+- #### Node installation on Ubuntu
+
+  You can install nodejs and npm easily with apt install, just run the following commands.
+
+      $ sudo apt install nodejs
+      $ sudo apt install npm
+
+- #### Other Operating Systems
+  You can find more information about the installation on the [official Node.js website](https://nodejs.org/) and the [official NPM website](https://npmjs.org/).
+
+  If the installation was successful, you should be able to run the following command.
+    
+      $ node --version
+      v12.18.2
+      
+      $ npm --version
+      6.14.8
+
+- #### For Yarn installation
+  After installing node, this project will need yarn too, so just run the following command.
```

Comment:
we actually don't require yarn.  I think we are using NPM by default.  Yarn is optional if you rather develop with yarn.

Reply:
PR is updated now.